__tests__/server/middlewares/cors.test.ts

```diff
@@ -0,0 +1,58 @@
+import { corsMiddleware } from '@server/middlewares';
+import express from 'express';
+import supertest from 'supertest';
+
+const app = express();
+app.use(corsMiddleware);
+
+app.get('', (req, res) => {
+  res.statusCode = 200;
+  res.end();
+});
+
+const request = supertest(app);
+
+beforeAll(async () => {
+  jest.doMock('config', () => {
```

Comment:
@ketanreliefapps what I tried to do there:

make sure that we can, during the call, exactly tell what the server.allowedOrigins should be, for the test

otherwise, it will just end up with the test config file

It does not work like that, I let you search for a solution

Also working in another branch with same issue, so I may come back to you with a solution

If you find something, you can tell me